manifests/rake.pp

```diff
@@ -14,6 +15,7 @@
     logoutput   => 'on_failure',
     refreshonly => true,
```

Comment:
This needs to be `false` when unless is set. Otherwise the whole trick still doesn't work.